src/status_im2/common/home/banner/view.cljs

```diff
@@ -33,12 +33,13 @@
        :blur-type     (theme/theme-value (if platform/ios? :light :xlight) :dark)
        :overlay-color (if open-sheet?
                         (colors/theme-colors colors/white colors/neutral-95-opa-70)
-                        (theme/theme-value nil colors/neutral-95-opa-70))}]]))
+                        (theme/theme-value nil colors/neutral-95-opa-70))}
+      children]]))
```

Comment:
@mohsen-ghafouri, If `banner-card-blur-layer` can accept multiple components as children, then we need to use a pattern like `[into [blur/view ...] children]` and change the function signature to be variadic, or at least that's the common convention.

But since `banner-card-blur-layer` only receives a child, I think we should rename `children` to `child` and avoid the unnecessary variadic signature.

Reply:
In other words, the word `children` implies it's a collection, but a collection can't directly be added to a view. We get an exception. So that's the reasoning behind using the word in the singular or changing the function signature and code to properly deal with multiple children.

Reply:
got it, thanks